packages/vite-plugin-checker/src/checkers/vueTsc/prepareVueTsc.ts

```diff
@@ -34,7 +41,7 @@ const textToReplace: { target: string; replacement: string }[] = [
     target: `function createProgram(rootNamesOrOptions, _options, _host, _oldProgram, _configFileParsingDiagnostics) {`,
     replacement: `function createProgram(rootNamesOrOptions, _options, _host, _oldProgram, _configFileParsingDiagnostics) { return require(${JSON.stringify(
       proxyPath
-    )}).createProgramProxy(...arguments);`,
+    )}).createProgram(...arguments);`,
```

Comment:
should be be `createProgramProxy` when patching `vue-tsc/out/proxy` as before and `createProgram` for `vue-tsc/out/index`.

Reply:
Of course, thanks 👍